packages/recommend-core/src/getTrendingFacets.ts

```diff
@@ -29,18 +29,14 @@ export type GetTrendingFacetsProps<TObject> = TrendingFacetsProps<TObject> &
 export function getTrendingFacets<TObject>({
   recommendClient,
   transformItems = (x) => x,
-  fallbackParameters,
   indexName,
   maxRecommendations,
-  queryParameters,
   threshold,
   facetName,
 }: GetTrendingFacetsProps<TObject>) {
```

Comment:
In the API client, the `TrendingFacetsQuery` type also needs to be adapted, because it still contains `queryParameters` and `fallbackParameters`.